src/docs/self-hosted/custom-ca-roots.mdx

```diff
@@ -12,7 +12,7 @@ The container's logs will have the output from `update-ca-certificates` right at
 
 ## Dependencies With Bundled Roots
 
-Some dependencies have opted to bundle their own CA roots and ignore the system CA roots. Where known, they have been configured to use the system roots. If something seems to ignore the system roots, [create an issue](https://github.com/getsentry/onpremise/issues/new?template=bug.yml) so it can be tracked down and fixed.
+Some dependencies have opted to bundle their own CA roots and ignore the system CA roots. Where known, they have been configured to use the system roots. If something seems to ignore the system roots, [create an issue](https://github.com/getsentry/self-hosted/issues/new?template=problem-report.yml) so it can be tracked down and fixed.
```

Comment:
Also picked off `bug.yml` &rarr; `problem-report.yml` to keep up with https://github.com/getsentry/self-hosted/pull/1159.